### ISSUE TITLE
Unnecessary "Invoking a constructor without parens and without arguments" error

### ISSUE THREAD

**Issue by mitar** [clutchski/coffeelint#421]
With `empty_constructor_needs_parens` enabled, for the following line:

```coffee
new OPERATIONS[operationSpec.type] operationSpec.field
```

I am getting this error:

```
Invoking a constructor without parens and without arguments.
```

This seems erroneous.

**Comment by AsaAyers** [comment]
I thought I recognized you :)

I was thinking this was a duplicate of #308, but I see the issue here is that you _are_ passing arguments. CoffeeLint just isn't recognizing those arguments.

**Comment by mitar** [comment]
Oh, I also had some memories I was reporting this already. :-)

**Comment by AsaAyers** [comment]
I went back and read that other issue again. It is actually the same. This morning when I read it I was thinking the other issue was about `camel_case_classes` and maybe CoffeeLint complains about `new foo()` vs `new Foo()`

**Comment by mitar** [comment]
Oh, true.
